how-to/how-to-add-telerik-ui-for-nativescript-package.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to Include the Telerik UI for NativeScript Package
+description: Tutorial on how to include the Telerik UI for NativeScript package to AppBuilder project
```

Comment:
You might want to benefit from the meta-description by including more keywords and expanding it a bit. Consider including the "pro" and "free" terms in here. Also, think about the description as the thing that is most likely to come up in the search results. Here is an example description:

A tutorial for adding Telerik UI for NativeScript to a mobile app in AppBuilder. You can use the UI controls by including the free limited edition or by downloading the complete package from your account and importing it in your app.